lib/shopify_app/session/storage_strategies/shop_storage_strategy.rb

```diff
@@ -0,0 +1,24 @@
+module ShopifyApp
+  module SessionStorage
+    class ShopStorageStrategy
```

Comment:
I think it's worth having specific tests for theses strategy classes.

Reply:
Hey @tylerball - these models don't exist in the gem - they are generated, and only exist in the subsequent project that is using the `shopify_app` gem. Do you have any suggestions on how we could test?

Reply:
Honestly there should have been one already for the shop session in the same way as the InMemoryStore but you should be able to test these strategies in the same way as the InMemoryStore. These models exist in our Gem, and all you need to do is define a TestClass within your tests that gets extended by your concern.

Reply:
Tests have been added.